includes/Modules/Analytics_4.php

```diff
@@ -128,6 +164,10 @@ public function register() {
 			function( $old_value, $new_value ) {
 				if ( $old_value['measurementID'] !== $new_value['measurementID'] ) {
 					$this->reset_data_available();
+
+					if ( Feature_Flags::enabled( 'newsKeyMetrics' ) ) {
```

Comment:
I've made the data availability state reset when the measurement ID changes, rather than the property ID, for consistency with the existing data availability state.